src/ckb_chain/actor.rs

```diff
@@ -104,29 +104,37 @@ impl Actor for CkbChainActor {
                     });
                 }
             }
-            SendTx(tx) => {
+            SendTx(tx, reply_port) => {
                 let rpc_url = state.config.rpc_url.clone();
                 tokio::task::block_in_place(move || {
                     let ckb_client = CkbRpcClient::new(&rpc_url);
-                    if let Err(err) = ckb_client.send_transaction(tx.data().into(), None) {
-                        //FIXME(yukang): RBF or duplicated transaction handling
-                        match err {
-                            RpcError::Rpc(e)
-                                if (e.code.code() == -1107 || e.code.code() == -1111) =>
-                            {
-                                log::warn!(
-                                    "[{}] transaction already in pool",
-                                    myself.get_name().unwrap_or_default()
-                                );
-                            }
-                            _ => {
-                                log::error!(
-                                    "[{}] send transaction failed: {:?}",
-                                    myself.get_name().unwrap_or_default(),
-                                    err
-                                );
+                    let result = match ckb_client.send_transaction(tx.data().into(), None) {
+                        Ok(_) => Ok(()),
```

Comment:
Do we need to wait the shutdown transaction is confirmed?

Reply:
I don't know. On the one hand, the fee may be so low that miners are never willing to pick up this closing transaction, thus this transaction may fail permanently. On the other hand, we have to wait indefinitely (due to network jitters etc) if we really want the transaction to be confirmed

Reply:
If the transaction is not picked by low fee, any partner can not have a chance to retry it by raising fee, since the transaction is signed by two partners?

Reply:
I am not really familiar with the transaction fee bumping techniques used in ckb. But the closing transaction created here is a valid transaction. But I think methods like https://bitcoinops.org/en/topics/cpfp/ can create enough incentives  for miner. So my previous conclusion of this transaction may fail permanently may be incorrect.